src/utils.ts

```diff
@@ -1,10 +1,17 @@
+import { VERSION as NG_VERSION } from '@angular/compiler';
 import * as ng from '@angular/compiler/src/expression_parser/ast';
 import { Lexer } from '@angular/compiler/src/expression_parser/lexer';
 import { Parser } from '@angular/compiler/src/expression_parser/parser';
 import { RawNGComment, RawNGSpan } from './types';
 
 const NG_PARSE_FAKE_LOCATION = 'angular-estree-parser';
 const NG_PARSE_TEMPLATE_BINDINGS_FAKE_PREFIX = 'NgEstreeParser';
+const NG_PARSE_FAKE_ABSOLUTE_OFFSET = 0;
+/* istanbul ignore next */
+const NG_PARSE_SHARED_PARAMS: readonly [string, number] =
+  NG_VERSION.major === '6' || NG_VERSION.major === '7'
+    ? ([NG_PARSE_FAKE_LOCATION] as any)
+    : ([NG_PARSE_FAKE_LOCATION, NG_PARSE_FAKE_ABSOLUTE_OFFSET] as const);
```

Comment:
I found that the `absoluteOffset` parameter was introduced in v8.2.0 (https://github.com/angular/angular/commit/f65db20c6d499b5cbaf6f3d3b2c118c791d06980), so we need to handle the special case for 8.0.0 ~ 8.1.x and test it in the CI as well.